README.md

```diff
@@ -193,6 +208,8 @@ The MySQL storage engine has a lower length limit for some properties of a tuple
 
 The core [OpenFGA](https://github.com/openfga/openfga) service has been in use by [Auth0 FGA](https://fga.dev) in production since December 2021.
 
+OpenFGA's Memory Adapter was built for development purposes only and is not recommended for a production environment, since it will not scale well for a large number of tuples.
```

Comment:
Drop the "since it will not scale well ..." bit. It's unnecessary. A datastore without persistence goes without saying for production usage.

or at least reword to something like

", because it is not designed for scalable queries and has no support for persistence."